infrastructure/application/services/hasura.ts

```diff
@@ -169,8 +168,10 @@ export const createHasuraService = async ({
   // TODO: bump awsx to 1.x to use the FargateService scaleConfig option to replace more verbose config below
   const hasuraScalingTarget = new aws.appautoscaling.Target("hasura-scaling-target", {
     // maxCapacity should consider compute power of the RDS instance which Hasura relies on
-    maxCapacity: parseInt(config.require("hasura-scaling-maximum")),
-    minCapacity: 1,
+    maxCapacity: parseInt(config.require("hasura-service-scaling-maximum")),
+    // minCapacity should reflect the baseline load expected
+    // see: https://hasura.io/docs/2.0/deployment/performance-tuning/#scalability
```

Comment:
note in particular their formula:

```
total_nodes = required_ccu / requests_per_node + backup_node
```

translation for our case: 'requests per node' capacity is ~ 500 concurrent users (doing heavy stuff like pulling down published flow data), so if we wanted to serve 2000 such users (which we may at some future juncture), by Hasura's count we should be running 5 nodes at minimum (4 + 1 for good measure)